ppr-ui/src/components/mhrTransfers/DeathCertificate.vue

```diff
@@ -0,0 +1,180 @@
+          <v-col cols="3" class="pr-6">
+            <label
+              class="generic-label"
+              for="deathCertificate"
+              :class="{ 'error-text': validateDeathCertificate && hasError(deathCertificateRef) }"
+            >
+              Death Certificate Number
+            </label>
+          </v-col>
+          <v-col cols="9">
+            <v-text-field
+              id="death-certificate-number"
+              v-model="deathCertificate"
+              ref="deathCertificateRef"
+              filled
+              :rules="deathCertificateRules"
+              label="Death Certificate Number"
+              data-test-id="death-certificate-number"
+            />
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col cols="3">
+            <label
+              class="generic-label"
+              for="date-of-death"
+              :class="{ 'error-text': validateDeathCertificate && !dateOfDeath }"
+            >
+              Date of Death
+            </label>
+          </v-col>
+          <v-col cols="9">
+            <date-picker
+              id="date-of-death"
+              clearable
+              ref="dateOfDeathRef"
+              title="Date of Death"
+              :errorMsg="validateDeathCertificate && !dateOfDeath ? 'Enter date of death' : ''"
+              :initialValue="dateOfDeath"
+              :key="Math.random()"
+              :maxDate="localTodayDate(maxDeathDate)"
+              @emitDate="dateOfDeath = $event"
+              @emitCancel="dateOfDeath = null"
+              @emitClear="dateOfDeath = null"
+              data-test-id="date-of-death"
+            />
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-spacer></v-spacer>
+          <v-col cols="9">
+            <v-checkbox
+              id="has-certificate-checkbox"
+              label="I have an original or certified copy of the death certificate, and confirm
+              that it was issued from Canada or the United States, and the name on
+              the death certificate matches the name displayed above exactly."
+              v-model="hasCertificate"
+              class="mt-0 pt-0 has-certificate-checkbox"
+              data-test-id="has-certificate-checkbox"
+            />
+          </v-col>
+        </v-row>
+      </v-form>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { DatePicker } from '@bcrs-shared-components/date-picker'
+import { useInputRules } from '@/composables'
+import { computed, defineComponent, reactive, ref, toRefs, watch } from '@vue/composition-api'
+import { useActions } from 'vuex-composition-helpers'
+import { FormIF, MhrRegistrationHomeOwnerIF } from '@/interfaces' // eslint-disable-line no-unused-vars
+import { localTodayDate } from '@/utils'
+
+export default defineComponent({
+  name: 'DeathCertificate',
+  props: {
+    deceasedOwner: {
+      type: Object as () => MhrRegistrationHomeOwnerIF,
+      default: null
+    }
```

Comment:
What information does this component require from the deceased HomeOwner?

Reply:
This component would set the `deathCertificateNumber` and `deathDateTime` for the deceased owner once valid

Reply:
For sure, i guess using whats existing, `editHomeOwner` it takes the owner and the groupId. 
That being said, i wouldn't copy the prop value to another local property, you will lose reactivity on the object. 

So when you go to set it, just spread it. ie `setSomeEditedHomeOwner({...props.homeOwner, deathCertNumber: someNewVal })` as an example.

Reply:
Ohhh interesting! I'll remove the local instance then. I'm able to manipulate the prop then set it though right?

Reply:
More or less. 
We never mutate the prop directly but spread is a means of shallow copying.